src/injected/components/details-dialog.tsx

```diff
@@ -142,15 +144,20 @@ export class DetailsDialog extends React.Component<DetailsDialogProps, DetailsDi
         );
     }
 
+    private renderCloseButton(): JSX.Element {
```

Comment:
should this be called `renderCloseIcon` instead?

Reply:
Yes, that would be more accurate. Will update.